app/frontend/actions/error.ts

```diff
@@ -0,0 +1,34 @@
+import debugLog from '../helpers/debugLog'
+import captureBySentry from '../helpers/captureBySentry'
+import {Store, State} from '../state'
+
+export default ({setState}: Store) => {
+  return {
+    // TODO: remove "options", as "setError" should not be responsible for setting other
+    // parts of the state
+    setError: (
+      state: State,
```

Comment:
are we sure we want to pass `state` as param if we are not using it? Now that we have typescript, we can afford variability in parameters

Reply:
I was thinking about this too, though I thought that it might be good to be explicit about the fact that all actions should receive "state". You can then avoid error when you add "action params" as first argument and are surprised why it does not work.